### ISSUE TITLE
Similar project

### ISSUE THREAD

**Issue by rolivares** [a-s-o/koa-docs#16]
Do you know any similar project to generate documentation using koa-joi-router or similar?, it appears this project is a little frozen.
thanks in advance

**Comment by miserylee** [comment]
I recommend this libaray which maintained by myself.
[koact](https://github.com/miserylee/koact)  It's a smart koa router manager.

There is a minify example application in the `/test` folder  [view test folder](https://github.com/miserylee/koact/tree/master/test)

it use [schema.io](https://github.com/miserylee/schema.io) libaray to validate request params and automatic generate api documents.

Here is the api doc preview 
![image](https://user-images.githubusercontent.com/16648966/76566868-a3b59c80-64e8-11ea-9d98-e4aa6174ec48.png)

The only problem is that the document of libaray is in Chinese, maybe hard to understand for you.

**Comment by rolivares** [comment]
yes it is !! I'd love to read chinese but I don't do it. ... I'l take a look any way ;) thanks
